server/index.js

```diff
@@ -6,19 +7,25 @@ const mongoose = require('mongoose');
 const PORT = 4000;
 
 app.use(cors());
-app.use(bodyParser.json);
+app.use(bodyParser.json());
 
-mongoose.connect('mongodb://127.0.0.1.27017/ecommerce', {
+mongoose.connect(process.env.DATABASE_URL, {
```

Comment:
```suggestion
mongoose.connect(process.env.DATABASE_URL || mongodb://127.0.0.1.27017/ecommerce, {
```
Worth adding in a fallback incase someone doesn't have an environment variable set up